### PR TITLE
Fixed how articles without images are handled

### PR DESCRIPTION
**Pull request by tjcarrel**

### CODE REVIEW

File: Cornell Sun/Cornell Sun/Cells/SearchCellNoImage.swift [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    
+    let insetConstant: CGFloat = 18
+    let offsetConstant: CGFloat = 12
+    let imageViewWidthHeight: CGFloat = 90
+    let timeLabelOffset: CGFloat = -8
+    
+    var post: PostObject? {
+        didSet {
+            if let post = post {
+                authorLabel.text = post.author[0].name
```
Comment (AAAstorga) [inline Cornell Sun/Cornell Sun/Cells/SearchCellNoImage.swift]: do `post.author.byline`. You might need to wait for Mindy to merge her changes @mindylou

File: Cornell Sun/Cornell Sun/Feed Section/ArticleSectionController.swift [hunk 2]
```diff
@@ -65,7 +65,11 @@ extension ArticleSectionController: BookmarkPressedDelegate, SharePressedDelegat
             return CGSize(width: width, height: height + 13)
 
         case .imageCell:
-            return CGSize(width: width, height: width / 1.92)
+            if(entry.featuredMediaImages.mediumLarge?.url == nil) {
+                return CGSize(width: 0, height: 0)
```
Comment (AAAstorga) [inline Cornell Sun/Cornell Sun/Feed Section/ArticleSectionController.swift]: return `.zero`

File: Cornell Sun/Cornell Sun/Search Section/SearchSectionController.swift [hunk 3]
```diff
@@ -66,9 +66,15 @@ class SearchSectionController: ListSectionController {
             return cell
         case .previewCell:
             // swiftlint:disable:next force_cast
-            let cell = collectionContext!.dequeueReusableCell(of: SearchCell.self, for: self, at: index) as! SearchCell
-            cell.post = entry
-            return cell
+            if(entry.featuredMediaImages.thumbnail?.url != nil) {
```
Comment (mindylou) [inline Cornell Sun/Cornell Sun/Search Section/SearchSectionController.swift]: remove parentheses around `if (....)`

File: Cornell Sun/Cornell Sun/Feed Section/ArticleSectionController.swift [hunk 4]
```diff
@@ -65,7 +65,11 @@ extension ArticleSectionController: BookmarkPressedDelegate, SharePressedDelegat
             return CGSize(width: width, height: height + 13)
 
         case .imageCell:
-            return CGSize(width: width, height: width / 1.92)
+            if(entry.featuredMediaImages.mediumLarge?.url == nil) {
```
Comment (mindylou) [inline Cornell Sun/Cornell Sun/Feed Section/ArticleSectionController.swift]: remove parentheses

File: Cornell Sun/Cornell Sun/Cells/SearchCellNoImage.swift [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func setupViews() {
+        self.backgroundColor = .white
+        addSubview(authorLabel)
+        addSubview(timeStampLabel)
+        addSubview(contentLabel)
+        
+        authorLabel.snp.makeConstraints { (make) in
```
Comment (mindylou) [inline Cornell Sun/Cornell Sun/Cells/SearchCellNoImage.swift]: remove parentheses

File: Cornell Sun/Cornell Sun/Cells/SearchCellNoImage.swift [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+        }
+        
+        timeStampLabel.snp.makeConstraints { (make) in
+            make.trailing.equalToSuperview().offset(-insetConstant)
+            make.top.equalToSuperview().inset(offsetConstant)
+        }
+        
+        contentLabel.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(insetConstant)
+            make.trailing.equalToSuperview().inset(insetConstant)
```
Comment (mindylou) [inline Cornell Sun/Cornell Sun/Cells/SearchCellNoImage.swift]: chain these constraints together `make.leading.trailing.equalToSuperview().inset(insetConstant)`

**Pull request merged**